Cargo.toml

```diff
@@ -10,7 +10,9 @@ documentation = "https://github.com/cryptape/cita-vm/blob/master/README.md"
 
 [dependencies]
 byteorder = "1.0"
+bytes = "0.4"
 cita_trie = "2.0"
+ckb-vm =  { git = "https://github.com/nervosnetwork/ckb-vm", features=["asm"] }
```

Comment:
Consider to add a commit hash to the dependency in case that ckb-vm makes some change not compatible.

Reply:
Sure.